rosa_getting_started_sts/rosa-sts-deleting-access-cluster.adoc

```diff
@@ -5,8 +5,12 @@ include::modules/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-Delete access to a {product-title} (ROSA) cluster using the `rosa` command-line.
+Access to a {product-title} (ROSA) cluster is controlled through the identity provider (IDP) accounts of users. To remove full access of a user to a cluster, you must delete the Identity Provider (IDP) account that was configured for the user.
```

Comment:
remove the word "full". It could be confusing with elevated permissions.

Reply:
Also, they shouldn't delete the IDP that they set up, but rather that configuration is done in the IdP.  A potential rephrase could be "To remove a user's access to a cluster , you must configure that within the Identity Provider (IDP) that has been set up for authentication", or something to that effect.

Reply:
Incorporated both the comments.